src/test/resources/testdata/adduser.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
we dont have this test yet so please remove this file. When test scenario is completely ready then raise PR.

Reply:
Ok.
Deleted the file "adduser.json" so that we can baseline the code as v1.0